examples/object_detection/dataflow/coco128.py

```diff
@@ -102,9 +106,14 @@ class COCO128Dataset(Dataset):
     classes = list(label_to_name.values())
     name_to_label = {v: k for k, v in enumerate(classes)}
 
-    def __init__(self, data_path, mode, buffer_size=None, return_yolo_dict=False):
+    def __init__(self, data_path, mode, buffer_size=64, download=False, return_yolo_dict=False):
         if not has_ultralytics:
             raise RuntimeError("To use Yolo models, please install ultralytics:\n\tpip install ultralytics")
+        if download == True:
+            check_det_dataset("coco128.yaml")
+            shutil.move("datasets/coco128", data_path)
+        elif not Path(data_path).exists():
+            raise RuntimeError("Dataset not found or corrupted. You can use download=True to download it")
```

Comment:
Cool! I also tried to use torchvision tool to download the dataset from their url as here: https://github.com/vfdev-5/ignite/blob/ea4bfdc6bcecb3eacd52d525b9bc5721bc943bcb/examples/object_detection/main.py#L491-L495

Happy to move that code here or use `check_det_dataset`, I let you decide.

Reply:
let's use yours as it doesn't increase the dependency on ultralytics

Reply:
OK, so let's remove the import and download arg.